app/javascript/controllers/podcast_planner_controller.js

```diff
@@ -0,0 +1,9 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
```

Comment:
i think i modeled this after the `allocations` controller in Augury, but obviously very pared down and simplified to what i needed it for, which is to update the dates "preview" in the view.